src/style.css

```diff
@@ -0,0 +1,110 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  font-weight: 400;
+  margin: 0;
+  line-height: 15px;
+}
+
+h1 {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  padding: 30px 14px 15px;
+  margin: 0 auto;
+  border: 1px solid #f3f3f3;
+}
+
+.product__wrapper {
+  margin-bottom: 41px;
+}
+
+.product__img {
+  width: 160px;
+  height: 134px;
+}
+
+.description {
+  max-width: 170px;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 14px;
+}
+
+.description__name {
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.rate {
+  margin-bottom: 22px;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.rate__stars {
+  display: flex;
+}
+
+.rate__star {
+  background: url("./images/star.svg") no-repeat center;
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+}
+
+.stars--1 .rate__star:first-child,
+.stars--2 .rate__star:nth-child(-n + 2),
+.stars--3 .rate__star:nth-child(-n + 3),
+.stars--4 .rate__star:nth-child(-n + 4),
+.stars--5 .rate__star:nth-child(-n + 5) {
+  background: url("./images/star-active.svg") no-repeat center;
+}
+
+.rate__link {
+  color: black;
+}
+
+.price {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  margin-bottom: 20px;
+}
+
+.price__amount {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.product__button {
+  color: #ffff;
+  font-size: 14px;
+  text-transform: uppercase;
+  padding: 12px 57px;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+}
+
+.product__button:hover {
+  display: block;
```

Comment:
if you need a display block then use it on the button itself

Reply:
Done